docs/proposal-config-changes.md

```diff
@@ -0,0 +1,731 @@
+# Proposal - changes to global and workflow configs
+
+These notes mainly address [cylc-flow #3422](https://github.com/cylc/cylc-flow/issues/3422).
+
+## General points
+
+### Terminology
+   * We should remove all references to `suite` in the settings (partly because
+     we're trying stop referring to "suites" and partly because some affected
+     settings are actually referring to the workflow server which is unclear).
+     * Note: "suite" is referred to through-out the code, tests and
+       documentation.
+   * Do we want to use `server` as shorthand for `workflow server`?
```

Comment:
> Explicit is better than implicit.
Tim Peters

Reply:
(We've now abandoned the term "workflow server")